src/transformers/integrations/integration_utils.py

```diff
@@ -692,10 +692,19 @@ def print_to_file(s):
 
 class WandbCallback(TrainerCallback):
     """
-    A [`TrainerCallback`] that logs metrics, media, model checkpoints to [Weight and Biases](https://www.wandb.com/).
+    A [`TrainerCallback`] that logs metrics, media, evals, and model checkpoints to [Weight and Biases](https://www.wandb.com/).
     """
 
-    def __init__(self):
+    def __init__(
```

Comment:
There should be a docstring for all these args. In particular `num_samples` and `freq` which aren't obvious